apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -66,7 +66,6 @@ type OpsRequestSpec struct {
 	// +patchStrategy=merge,retainKeys
 	// +listType=map
 	// +listMapKey=componentName
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="forbidden to update spec.volumeExpansion"
```

Comment:
and why is remove immutable check fix the bug? expect UT updates.

Reply:
s/i/d of Quantity is private, only provided the MustParse function to create it . but this function will update s/i/d firstly.

Reply:
when creates the opsRequest by `yaml`, it only sets the value of `s or i or d` of Quantity struct.  and opsRequest controller will update labels/finalizers or more, then the  Quantity will sync the value in `s, i,d`